solidity/random-beacon/deploy/03_deploy_random_beacon.ts

```diff
@@ -0,0 +1,83 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { getNamedAccounts, deployments, helpers } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const T = await deployments.get("T")
+  const TokenStaking = await deployments.get("TokenStaking")
+  const ReimbursementPool = await deployments.get("ReimbursementPool")
+  const BeaconSortitionPool = await deployments.get("BeaconSortitionPool")
+
+  const BLS = await deployments.deploy("BLS", {
+    from: deployer,
+    log: true,
+  })
+
+  const BeaconAuthorization = await deployments.deploy("BeaconAuthorization", {
+    from: deployer,
+    log: true,
+  })
+
+  const BeaconDkg = await deployments.deploy("BeaconDkg", {
+    from: deployer,
+    log: true,
+  })
+
+  const BeaconInactivity = await deployments.deploy("BeaconInactivity", {
+    from: deployer,
+    log: true,
+  })
+
+  const BeaconDkgValidator = await deployments.deploy("BeaconDkgValidator", {
```

Comment:
I think `BeaconDkgValidator` deserves its own script because it is not a library compared to other dependencies like `BeaconDkg, BeaconInactivity etc`.

Reply:
[846836e](https://github.com/keep-network/keep-core/pull/3018/commits/846836e78e6937d9666d5792b1dd27023b84e0e5)